packages/zowe-explorer/src/configuration/Profiles.ts

```diff
@@ -1210,4 +1211,20 @@ export class Profiles extends ProfilesCache {
         };
         return [qpItemAll, qpItemCurrent];
     }
+
+    public static extenderTypeReady = new Map();
+
+    public _resolveTypePromise(extenderType: string): void {
```

Comment:
Why do we prefix this with `_`?
Should we make it private/protected?
If not possible... Should we make this `@internal`? 😋